internal/cmd/login.go

```diff
@@ -24,57 +23,42 @@ import (
 	"github.com/infrahq/infra/internal/certs"
 	"github.com/infrahq/infra/internal/cmd/types"
 	"github.com/infrahq/infra/internal/format"
-	"github.com/infrahq/infra/internal/generate"
 	"github.com/infrahq/infra/internal/logging"
 )
 
 type loginCmdOptions struct {
 	Server             string
-	AccessKey          string
```

Comment:
The access key story will likely need to be revisited. The CLI currently can take access key from an environment variable but requires an existing configuration. This is currently done by setting `INFRA_SERVER` and `INFRA_ACCESS_KEY` followed by `infra login` which uses the environment variables to set up these configurations. By removing the access key login option, we're removing authn with access key.

Now the workflow described above isn't ideal since `infra login` should not be necessary when using an access key but the goal of using an access key with the Infra CLI should still be valid.

Reply:
This is a fantastic catch, thanks! I've restored logging in with access keys since this is a simple way to authenticate as a non-human user.